docs/content/integrations/embedded-elt/dlt.mdx

```diff
@@ -256,29 +248,26 @@ def example_dlt_source():
     return example_resource
 
 
-class CustomDagsterDltTranslator(DagsterDltTranslator):
-    def get_asset_key(self, resource: DagsterDltResource) -> AssetKey:
-        """Overrides asset key to be the dlt resource name."""
-        return AssetKey(f"{resource.name}")
-
-    def get_deps_asset_keys(self, resource: DagsterDltResource) -> Iterable[AssetKey]:
-        """Overrides upstream asset key to be a single source asset."""
-        return [AssetKey("common_upstream_dlt_dependency")]
-
-
-@dlt_assets(
+dlt_source = example_dlt_source()
+dlt_pipeline = dlt.pipeline(
+    pipeline_name="example_pipeline_name",
+    dataset_name="example_dataset_name",
+    destination="snowflake",
+    progress="log",
+)
+source_asset_key = "common_upstream_dlt_dependency"
+RunDlt(
     name="example_dlt_assets",
-    dlt_source=example_dlt_source(),
-    dlt_pipeline=dlt.pipeline(
-        pipeline_name="example_pipeline_name",
-        dataset_name="example_dataset_name",
-        destination="snowflake",
-        progress="log",
-    ),
-    dagster_dlt_translator=CustomDagsterDltTranslator(),
+    dlt_source=dlt_source,
+    dlt_pipeline=dlt_pipeline,
+    specs=[
+        RunDlt.default_spec(dlt_source, dlt_pipeline, dlt_resource)._replace(
+            key=dlt_resource.name,  # overrides asset key to be resource name
+            deps=[source_asset_key],  # overrides upstream to be single source asset
+        )
+        for dlt_resource in dlt_source.resources.values()
+    ],
```

Comment:
This example is substantially simpler.

* No need to write the body of function, which is nice.
* No custom translator. While you have to know about the  `RunDlt.default_spec` this is to me, much much easier to compose and debug. The specs attached the computation are constructed before passing it in. It is very obvious what should happen. If you write a separate unit test easily that just deals with specs, or inspect them in a debugger/interpreter prior to passing in. The mental model is that specs are immutable once passed into a computation, rather than mutation after the fact managed by an opaque "translation" process.